tests/integration/test_bayesian_optimization.py

```diff
@@ -166,28 +168,32 @@ def build_model(data: Dataset) -> GaussianProcessRegression:
     initial_data = observer(initial_query_points)
     model = build_model(initial_data)
 
-    dataset = (
-        BayesianOptimizer(observer, search_space)
-        .optimize(num_steps, initial_data, model, acquisition_rule)
-        .try_get_final_dataset()
-    )
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        summary_writer = tf.summary.create_file_writer(tmpdirname)
+        with tensorboard_writer(summary_writer):
```

Comment:
Does this need to happen for each integration test we are running? I don't actually know if there is any noticeable overhead for additional I/O. But if there is, perhaps having a separate integ test that just tests tensorboard logging once would suffice?

Reply:
There's no significant time overhead (it just logs a small number of values to file)